validator/lib/validateTileset.js

```diff
@@ -95,31 +97,29 @@ function validateTileHierarchy(root, tilesetDirectory) {
         }
 
         if (defined(content) && defined(content.boundingVolume)) {
-            var contentRegion = content.boundingVolume.region;
-            var contentSphere = content.boundingVolume.sphere;
-            var contentBox = content.boundingVolume.box;
-            var tileRegion = tile.boundingVolume.region;
-            var tileSphere = tile.boundingVolume.sphere;
-            var tileBox = tile.boundingVolume.box;
-
-            if (defined(contentRegion) && defined(tileRegion) && !regionInsideRegion(contentRegion, tileRegion)) {
-                return 'content region [' + contentRegion + '] is not within tile region + [' + tileRegion + ']';
+            var outerTransform = Matrix4.IDENTITY;
+            if (defined(tile.transform)) {
+                outerTransform = Matrix4.fromArray(tile.transform);
             }
-
-            if (defined(contentSphere) && defined(tileSphere) && !sphereInsideSphere(contentSphere, tileSphere)) {
-                return 'content sphere [' + contentSphere + '] is not within tile sphere + [' + tileSphere + ']';
+            var innerTransform = Matrix4.IDENTITY;
+            var message = checkBoundingVolume(content.boundingVolume, tile.boundingVolume, innerTransform, outerTransform);
```

Comment:
I'm getting some warnings since `outerTransform`, `innerTransform` and `message` are declared multiple times in the same function scope. Can you declare these above, like:

```
        var outerTransform;
        var innerTransform
        var mesage;

        if (defined(content) && defined(content.boundingVolume)) {
            outerTransform = Matrix4.IDENTITY;
            if (defined(tile.transform)) {
                outerTransform = Matrix4.fromArray(tile.transform);
            }
            innerTransform = Matrix4.IDENTITY;
            message = checkBoundingVolume(content.boundingVolume, tile.boundingVolume, innerTransform, outerTransform);
            if (defined(message)) {
                return 'content bounding volume is not within tile bounding volume: ' + message;
            }
        }
```